Used to provide correct OpenLDAP include directory and to remove bogus 
OpenLDAP lib directory, so that pkglint doesn't complain.

--- python-ldap-2.4.10/setup.cfg.orig	2013-05-01 14:47:08.110553898 -0700
+++ python-ldap-2.4.10/setup.cfg	2013-05-01 14:47:49.755987482 -0700
@@ -1,6 +1,6 @@
 [_ldap]
-library_dirs = /opt/openldap-RE24/lib /usr/lib
-include_dirs = /opt/openldap-RE24/include /usr/include/sasl /usr/include
+library_dirs =
+include_dirs = /usr/include/openldap /usr/include/sasl /usr/include
 defines = HAVE_SASL HAVE_TLS HAVE_LIBLDAP_R
 extra_compile_args = 
 extra_objects = 
